### PR TITLE
Update to the latest bridge.

### PR DESCRIPTION
**Pull request by pgavlin**
And add a .NET SDK.

### CODE REVIEW

**Comment by pgavlin** [comment]
cc @mikhailshilkov

File: sdk/dotnet/Appengine/Application.cs [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        public ApplicationUrlDispatchRulesGetArgs()
+        {
+        }
+    }
+    }
+
+    namespace Outputs
+    {
+
+    [OutputType]
```
Comment (CyrusNajmabadi) [inline sdk/dotnet/Appengine/Application.cs]: indentation here makes me sad :'(

**Comment by CyrusNajmabadi** [comment]
Are you updating these libs to **also** publish?  or is that coming in a separate PR?

File: sdk/dotnet/Appengine/Application.cs (hunk 1 above)
Comment (pgavlin) [reply sdk/dotnet/Appengine/Application.cs]: Yeah, it's on my list of things to improve :/

**Comment by pgavlin** [comment]
> Are you updating these libs to also publish? or is that coming in a separate PR?

https://github.com/pulumi/scripts/pull/89

**Pull request merged**